dev/com.ibm.jbatch.container/transformed10.bnd

```diff
@@ -0,0 +1,18 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/transform10.props
+
+Bundle-Name: Java Batch RI core runtime implementation Jakarta
+Bundle-SymbolicName: com.ibm.jbatch.container.jakarta.ee10
+Bundle-Description: Java Batch RI core runtime implementation; Jakarta Enabled
+
+#-includeresource: \
+#  @${repo;io.openliberty.jbatch.internal.jakarta.ee10}!/com/ibm/jbatch/container/persistence/jpa/*.class
```

Comment:
Obviously the projects are missing that you are referencing in the bnd files in this project.  You will need to create them.